packages/components/components/elvis-modal/src/react/styledComponents.tsx

```diff
@@ -4,27 +4,9 @@ import styled, { css, keyframes } from 'styled-components';
 
 const mobileMax = '767px';
 const desktopMin = '1024px';
-const modalBorderRadius = '8px';
-const modalMaxWidth = '704px';
 
-const modalMobilePadding = '32px 32px 24px 32px';
-const modalMobileTitlePaddingTop = '8px';
-const modalMobileTitlePaddingBottom = '8px';
 const modalMobileIllustrationHeight = '40vh';
 
-const modalTabletPadding = '40px';
-const modalTabletIllustrationHeight = '28vh';
-const modalTabletTitlePaddingBottom = '32px';
-const modalTabletMaxWidth = '500px';
-
-const modalTabletWithIllustrationTitlePaddingBottom = '16px';
-
-const modalDesktopPadding = '48px';
-const modalDesktopTitlePaddingBottom = '32px';
-
-const modalDesktopWithIllustrationPadding = '80px';
-const modalDesktopWithIllustrationTitlePaddingBottom = '24px';
```

Comment:
These have just been inlined, as they more or less were all only used in one place.